controllers/release/release_adapter.go

```diff
@@ -210,6 +214,45 @@ func (a *Adapter) getApplicationSnapshot() (*appstudioshared.ApplicationSnapshot
 	return applicationSnapshot, nil
 }
 
+// getEnabledTargetReleaseLink returns the ReleaseLink targeted by ReleaseLink in the Release being processed.
+// Only ReleaseLinks with the auto-release label set to true (or missing the label, which we treat the same as
+// having the label and it being set to true) will be searched for. If a matching ReleaseLink is not found or
+// the List operation fails, an error will be returned.
+func (a *Adapter) getEnabledTargetReleaseLink() (*v1alpha1.ReleaseLink, error) {
```

Comment:
Maybe change it to `getActiveTargetReleaseLink`? For some reason `enabled` sounds weird to me. Leaving the decision to you as you are the native speaker.

Reply:
I think English wise they mean about the same so if active sounds better to you, I am fine with that. Changed